ufo/ssh_client.py

```diff
@@ -24,6 +33,17 @@ class SSHClient(paramiko.SSHClient):
       }
 
   def connect(self, proxy_server):
+    """Establish a connection with the target proxy server.
+    Args:
```

Comment:
Just for parity's sake, please add the new lines between the top level docstring, args, returns, and raises sections.